source/elements/oneMKL/source/domains/dft/config_params/data_layouts.rst

```diff
@@ -0,0 +1,172 @@
+    :label: eq_idx_data_layout
+
+of the corresponding data container (``sycl::buffer`` or USM pointer) provided
+to the compute function, the data type of which is (possibly implicitly
+re-interpreted) as documented in the
+:ref:`table<onemkl_dft_config_data_element_types>` below. In the index
+expression :eq:`eq_idx_data_layout`,
+
+- :math:`s_j`, :math:`0 \leq j \leq d` represents the :ref:`generalized strides and offset<onemkl_dft_fwd_bwd_strides>` within :math:`d`-dimensional data sets in forward (resp. backward) domain, in number of elements of the :ref:`relevant elementary data type<onemkl_dft_config_data_element_types>`;
+
+- :math:`l` represents the :ref:`distance<onemkl_dft_fwd_bwd_distance>` between successive :math:`d`-dimensional data sets in forward (resp. backward) domain, in number of elements of the :ref:`relevant elementary data type<onemkl_dft_config_data_element_types>`;
+
+.. container:: section
+
+   .. _onemkl_dft_config_data_element_types:
+
+   .. rubric:: Elementary data type assumed by descriptors depending on their type and configuration (precision matching the :ref:`descriptor's<onemkl_dft_enum_precision>`)
+
+   .. list-table::
+        :header-rows: 1
+        :class: longtable
+
+        * -   Descriptor configuration
+          -   Element type of forward data
+          -   Element type of backward data
+        * -   ``COMPLEX`` template :ref:`onemkl_dft_enum_domain` and :ref:`onemkl_dft_complex_storage_complex_complex`
+          -   Complex
+          -   Complex
+        * -   ``COMPLEX`` template :ref:`onemkl_dft_enum_domain` and :ref:`onemkl_dft_complex_storage_real_real`
+          -   Real
+          -   Real
+        * -   ``REAL`` template :ref:`onemkl_dft_enum_domain`, :ref:`onemkl_dft_real_storage_real_real` and :ref:`onemkl_dft_conjugate_even_storage_complex_complex`
+          -   Real
+          -   Complex
+
+
+.. _onemkl_dft_fwd_bwd_strides:
+
+.. rubric:: FWD_STRIDE and BWD_STRIDES
+
+The values :math:`s_0, s_1, \dots, s_d` in :eq:`eq_idx_data_layout` here above
+are communicated as elements, in that order, of a :math:`(d+1)`-long
+``std::vector<std::int64_t>`` value for the configuration parameter
+``config_param::FWD_STRIDES`` (resp. ``config_param::BWD_STRIDES``). The element
+:math:`s_0` represents an absolute offset (or "displacement") in the data set(s)
+while the subsequent element(s) :math:`s_j\ (j > 0)` are generalized strides.
+
+The default values set for the forward and backward strides correspond to the
+data layout configurations for unbatched, in-place transforms using unit stride
+along the leading dimension with no offset (and minimal padding in forward
+domain in case of descriptors of ``REAL`` template :ref:`onemkl_dft_enum_domain`,
+aligning with the :ref:`requirements for in-place
+transforms<onemkl_dft_data_layout_requirements>`). In other words, the default
+values are :math:`s_0 = 0`, :math:`s_d = 1` and, for multi-dimensional
```

Comment:
```suggestion
values are :math:`s_0 = 0`, :math:`s_d = 1` and, for :math:`d`-dimensional
```

Reply:
changed it to $d$-dimensional DFTs with $d > 1$.